frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -94,6 +89,93 @@ export const createDeploymentConfig = (
     });
   });
 
+  return { labels, podLabels, volumes, volumeMounts };
+};
+
+export const createDeployment = (
+  formData: DeployImageFormData,
+  imageStreamName: string,
+  dryRun: boolean,
+): Promise<K8sResourceKind> => {
+  const {
+    project: { name: namespace },
+    name,
+    isi: { ports },
+    deployment: { env, replicas },
+    labels: userLabels,
+    limits: { cpu, memory },
+  } = formData;
+
+  const { labels, podLabels, volumes, volumeMounts } = getMetadata(formData);
+
+  const deployment = {
+    kind: 'Deployment',
+    apiVersion: 'apps/v1',
+    metadata: {
+      name,
+      namespace,
+      labels,
+      annotations,
```

Comment:
You should add an image change trigger with the `image.openshift.io/triggers` annotation

Reply:
https://docs.openshift.com/container-platform/3.11/dev_guide/managing_images.html#image-stream-kubernetes-resources

Reply:
can we use triggers in Deployment?

Reply:
> can we use triggers in Deployment?

Yes! But you need to add the annotation

Reply:
@gijohn, @sbose78 mentioned this the other day in our slack as well.
https://docs.openshift.com/container-platform/3.11/dev_guide/managing_images.html#using-is-with-k8s